src/views/GameListView.jsx

```diff
@@ -79,8 +81,21 @@ const Content = styled.div`
 
 const GameListView = () => {
   const { t } = useTranslation();
+  const NULL_SEARCH_TERM = "";
 
   const [gameList, setGameList] = React.useState([]);
+  const [searchTerm, setSearchTerm] = React.useState(NULL_SEARCH_TERM);
+
+  const handleChange = (event) => {
+    setSearchTerm(event.target.value);
+  };
+
+  const filteredGameList = React.useMemo(() => {
+    return gameList.filter((game) =>
+      searchTerm === NULL_SEARCH_TERM ||
+      game.defaultName.toLowerCase().includes(searchTerm.toLowerCase())
```

Comment:
Can you also handle diacritics? (See this [link](https://javascript.plainenglish.io/not-so-obvious-removal-of-diacritics-in-javascript-explained-and-done-right-52f4aeb3c85))